xontrib_term_integrations/utils.py

```diff
@@ -299,3 +300,23 @@ def set_user_var(
         write_tmux_cmd(term_osc_cmd(user_var))
     else:
         write_osc_cmd(user_var)
+
+
+def set_user_var_fn(var: str, val: Annotated[Optional[str], Arg(nargs="?")] = ""):
+    """Sets a terminal pane's user `Variable` to a given `Value`
```

Comment:
you can check why this error is raised by ruff or silence the error by using `#noqa`

Reply:
I've tried to google the ruff error code F722, but could only quickly find some references to regex strings and how it's false positive, thought you might know, otherwise I'll just silence it